src/TRestGeant4Metadata.cxx

```diff
@@ -219,6 +219,16 @@
 ///     <generator type="volume" shape="box" size="(10,20,20)" position="(0,0,5)" > ... </generator>
 /// \endcode
 ///
+/// * **cosmic**: This is a special type of generator that is used to simulate cosmic particles.
```

Comment:
Since all particles are directed towards the detector, there will be some biasing. How then we should renormalise the number of particles launched to understand how much equivalent will be that simulation in the real world?

Reply:
This is a good question, I think the equivalent surface would be `\pi * R^2` where `R` is the radius of the sphere used for sampling, which equals the diagonal of the world box. I will add a method to retrieve this and update the readme.